src/components/Icons/CheckBoxIcon.tsx

```diff
@@ -0,0 +1,38 @@
+import { ReactElement } from 'react';
+import cn from 'classnames';
+
+interface Props {
+  checked: boolean;
+}
+
+export const CheckBoxIcon = ({ checked }: Props): ReactElement => {
+  const iconClass = 'w-[36px] h-[36px] flex items-center justify-center';
+
+  return (
+    <div className={cn(iconClass)}>
+      <div className="group cursor-pointer justify-center text-center">
+        {checked ? (
+          <svg className="duration-300" width="37" height="37" viewBox="0 0 37 37" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
These SVGs should be imported and used within img tags - import img from '../assets/..'  <img src={img}>